comp/core/secrets/secretsimpl/refresh.tmpl

```diff
@@ -1,6 +1,6 @@
-=== Secret refresh stats ===
-Number of secrets refreshed: {{ len .Handles }}
-Secrets handle refreshed:
+=== Secret reload stats ===
+Number of secrets reloaded: {{ len .Handles }}
+Secrets handle reloaded:
```

Comment:
Looking back at it it seems weird that we're talking about reload/refresh here.

WDYT about:
```suggestion
=== Secret stats ===
Number of secrets: {{ len .Handles }}
Secrets handle:
```